alibi/explainers/cfproto.py

```diff
@@ -17,7 +17,16 @@
 logger = logging.getLogger(__name__)
 
 
-class CounterFactualProto(Explainer, FitMixin):
+def CounterFactualProto(*args, **kwargs):
+    # TODO: remove this function in an upcoming release
```

Comment:
Annoyingly having an empty docstring here doesn't make `apidoc` ignore the `CounterFactualProto` method. It probably makes sense to add a deprecation warning to the docstring instead?